### DISCUSSION TITLE
[Poll] Use `.jsx` extension for JSX files

### DISCUSSION THREAD

**Discussion by marlonmarcello** [wethegit/corgi#41] [Polls]
Vite requires files that contain JSX to have the `.jsx` extension. Which we updated all our libraries to. I think we should do the same thing in Corgi to stay consistent. Also easier to move components across bundlers.
It also makes sense. Those files require an extra transpilation step and they are indeed JSX components.

**Comment by NicholasLancey** [comment]
I don't have a major opinion on this, would just be nice to have a standard to stick to!

**Comment by marlonmarcello** [comment]
Vote ended.
Result: **Yes**
We'll be using `.jsx` for JSX files from now on.
